docs/api/php_api/php_api.md

```diff
@@ -48,21 +45,29 @@ The full list of available services covers:
 - [UserPreferenceService](https://github.com/ibexa/core/blob/main/src/contracts/Repository/UserPreferenceService.php)
 - [UserService](https://github.com/ibexa/core/blob/main/src/contracts/Repository/UserService.php)
 
+You can access the PHP API by injecting relevant services into your code.
+
+- Using [auto-wiring]([[=symfony_doc=]]/service_container/autowiring.html), and the service classname in the `Ibexa\Contracts` namespace (see `bin/console debug:autowiring | grep Ibexa.Contracts`).
+- Using [service parameters]([[=symfony_doc=]]/service_container.html#service-parameters), and the service aliases (see `bin/console debug:autowiring | grep ibexa.api`).
+- Using Repository's `get[ServiceName]()` methods: [`Repository::getContentService()`](https://github.com/ibexa/core/blob/v4.6.6/src/contracts/Repository/Repository.php#L46), [`getUserService()`](https://github.com/ibexa/core/blob/v4.6.6/src/contracts/Repository/Repository.php#L111), etc.
+  (Prefer injecting several Repository's dedicated services instead of the whole Repository if the Repository itself is not needed.)
+- (Not recommended) Using `Ibexa\Bundle\Core\Controller::getRepository()` by extending it from your [custom controller](controllers.md). (Prefer dependency injection.)
+
```

Comment:
```suggestion
You can access the PHP API by injecting relevant services into your code:

- By using [auto-wiring]([[=symfony_doc=]]/service_container/autowiring.html), and the service classname in the `Ibexa\Contracts` namespace (see `bin/console debug:autowiring | grep Ibexa.Contracts`).
- By using [service parameters]([[=symfony_doc=]]/service_container.html#service-parameters), and service aliases (see `bin/console debug:autowiring | grep ibexa.api`).
- By using the Repository's `get[ServiceName]()` methods: [`Repository::getContentService()`](https://github.com/ibexa/core/blob/v4.6.6/src/contracts/Repository/Repository.php#L46), [`getUserService()`](https://github.com/ibexa/core/blob/v4.6.6/src/contracts/Repository/Repository.php#L111), etc.
  (Prefer injecting several Repository's dedicated services instead of the whole Repository if the Repository itself is not needed.)

!!! note 

    You can also access the PHP API with `Ibexa\Bundle\Core\Controller::getRepository()` by extending it from your [custom controller](controllers.md), but such approach is not recommended, and you should prefer dependency injection.
```

Reply:
I would avoid using a "you" and depersonalize this. To get closer to the idea of "someone else wrote it this way, you won't, you might even update it"

```suggestion
You can access the PHP API by injecting relevant services into your code:

- By using [auto-wiring]([[=symfony_doc=]]/service_container/autowiring.html), and the service classname in the `Ibexa\Contracts` namespace (see `bin/console debug:autowiring | grep Ibexa.Contracts`).
- By using [service parameters]([[=symfony_doc=]]/service_container.html#service-parameters), and service aliases (see `bin/console debug:autowiring | grep ibexa.api`).
- By using the Repository's `get[ServiceName]()` methods: [`Repository::getContentService()`](https://github.com/ibexa/core/blob/v4.6.6/src/contracts/Repository/Repository.php#L46), [`getUserService()`](https://github.com/ibexa/core/blob/v4.6.6/src/contracts/Repository/Repository.php#L111), etc.
  (Prefer injecting several Repository's dedicated services instead of the whole Repository if the Repository itself is not needed.)

!!! caution 

    The PHP API's services can be accessed with `Ibexa\Bundle\Core\Controller::getRepository()` by extending it from a [custom controller](controllers.md), but such approach is not recommended, and you should prefer dependency injection.
```